geth/common/types.go

```diff
@@ -88,6 +88,12 @@ type NodeManager interface {
 	// RPCClient exposes reference to RPC client connected to the running node
```

Comment:
We should think about removing this method then.

Reply:
Yes, but first we must find out which areas use this and their context, to understand how the change affects the code.

Reply:
Sure. It can be done even later as part of some refactoring.